Dockerfile

```diff
@@ -24,4 +34,13 @@ RUN set -e \
  && SECRET_KEY=temp HOOVER_DB='postgresql://search:search@search-pg:5432/search' ./manage.py collectstatic --noinput \
  && chmod +x /wait
 
-CMD ./runserver
+ENV USER_NAME $UNAME
+ENV DATA_DIR /opt/hoover/search
+
+mkdir -p /opt/hoover/metrics
```

Comment:
check your tests...
Error response from daemon: Dockerfile parse error line 40: unknown instruction: MKDIR

these should be `RUN mkdir ...` and `RUN chown ...`